app/src/main/java/com/amaze/filemanager/ui/fragments/preference_fragments/BookmarksPrefsFragment.kt

```diff
@@ -36,74 +34,65 @@ import com.amaze.filemanager.database.UtilsHandler
 import com.amaze.filemanager.database.models.OperationData
 import com.amaze.filemanager.databinding.DialogTwoedittextsBinding
 import com.amaze.filemanager.filesystem.files.FileUtils
-import com.amaze.filemanager.ui.activities.superclasses.ThemedActivity
 import com.amaze.filemanager.ui.views.preference.PathSwitchPreference
 import com.amaze.filemanager.utils.DataUtils
 import com.amaze.filemanager.utils.SimpleTextWatcher
-import java.util.*
+import java.util.HashMap
 
-/** @author Emmanuel on 17/4/2017, at 22:49.
- */
-class FoldersPref : PreferenceFragmentCompat(), Preference.OnPreferenceClickListener {
+class BookmarksPrefsFragment : BasePrefsFragment() {
+    override val title = R.string.show_bookmarks_pref
+
+    companion object {
+        private val dataUtils = DataUtils.getInstance()!!
+    }
 
-    private var sharedPrefs: SharedPreferences? = null
     private val position: MutableMap<Preference, Int> = HashMap()
-    private var dataUtils: DataUtils? = null
-    private var utilsHandler: UtilsHandler? = null
+    private var bookmarksList: PreferenceCategory? = null
```

Comment:
This should probably be `lateinit var`

Reply:
Same as above: `findPreference` returns `Preference?`

Reply:
Makes sense, then leave it as is, and use "?" on every usage.